twine.gemspec

```diff
@@ -5,7 +5,7 @@ Gem::Specification.new do |s|
   s.name         = "twine"
   s.version      = Twine::VERSION
   s.date         = Time.now.strftime('%Y-%m-%d')
-  s.summary      = "Manage strings and their translations for your iOS and Android projects."
+  s.summary      = "Manage strings and their translations for your iOS, Android and other platform projects."
```

Comment:
I think I prefer:

`"Manage strings and their translations for iOS, Android, and other projects."`